DataFormats/PatCandidates/src/classes_def_objects.xml

```diff
@@ -275,7 +275,8 @@
    <version ClassVersion="11" checksum="1239840459"/>
   </class>
 
-  <class name="pat::PackedCandidate" ClassVersion="18">
+  <class name="pat::PackedCandidate" ClassVersion="19">
```

Comment:
this will have to become 21 (19 and 20 are already used in 75X and 76X).
check after that if 75X checksum will agree with this version.